.babelrc

```diff
@@ -35,6 +35,12 @@
       "plugins": [
         "./build/use-lodash-es"
       ]
+    },
+    "rollup": {
+      "plugins": [
+        "./build/use-lodash-es",
+        "external-helpers"
```

Comment:
It doesn't try to polyfill everything. Used is only what added by plugins in .babelrc. And it looks like runtime adds `core-js` stuff under the hood. And it's common js afaik.